OSDOCS-17075_e#Cluster updates CQAs (#115163)
openshift/openshift-docs: modules/update-etcd-backup.adoc

[id="update-etcd-backup_{context}"]
= etcd backups before cluster updates

[role="_abstract"]

> Create etcd backups before you update clusters to preserve your cluster state and to enable disaster recovery if the update fails and the cluster becomes unrecoverable.

Reviewer: ```suggestion
Create etcd backups before you update clusters to preserve your cluster state and to enable disaster recovery.
```

> etcd backups record the state of your cluster and all of its resource objects. You can use backups to attempt restoring the state of a cluster in disaster scenarios where you cannot recover a cluster in its currently dysfunctional state.

Reviewer: ```suggestion
etcd backups record the state of your cluster and all of its resource objects. You can use backups to try to restore the state of a cluster when the cluster has become unrecoverable.
```
IDK if this was AI output, but it's verbose and the sentences are long. Suggest simplifying.
